code/modules/mob/living/carbon/human/combat.dm

```diff
@@ -29,6 +29,16 @@
 /mob/living/carbon/human/disarm_mob(mob/living/target)
 	add_logs(src, target, "disarmed", admin = (src.ckey && target.ckey) ? TRUE : FALSE) //Only add this to the server logs if both mobs were controlled by player
 
+	var/datum/organ/external/S = target.get_organ(src.zone_sel.selecting)
+	if(src.zone_sel.selecting == "mouth" && !(S.status & ORGAN_DESTROYED) && ishuman(target))
+		var/mob/living/carbon/human/T = target
+		T.forcesay("")
```

Comment:
I really thought about that, but it would make them dash if they were saying nothing at all and I really didn't see a solution.  Any suggestions?